spec/javascripts/reducers/project_spec.js

```diff
@@ -0,0 +1,36 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/project";
+
+describe("Project Reducer",() => {
+  describe("RECEIVE_PROJECT",() => {
+    it("returns data from action",() => {
+      const state = {};
+
+      const action = {
+        type: actionsTypes.RECEIVE_PROJECT,
+        data: { id: 1 }
+      };
+
+      expect(reducer(state, action)).toEqual(action.data);
+    });
+  });
+
+  describe("ADD_LABEL_TO_PROJECT",() => {
+    it("adds new label to labels",() => {
+      const state = { labels: [] };
```

Comment:
What do you think about having a case where there is already some label to test that it'll append the list?